### PR TITLE
fixes broken link in docs

### PR DESCRIPTION
**Pull request by conradkdotcom**
This PR fixes a broken link in the docs. Nothing much but stumbled upon this when getting started with Mesos.

Docs are very nice btw, thanks ! 😃

### CODE REVIEW

**Comment by bbannier** [comment]
Thanks for the patch @conradkdotcom!

Grepping over docs tree I see there are in total four mentions of `(containerizer.md)`. It would be great if you could clean up all of them in one swoop.

**Comment by conradkdotcom** [comment]
@bbannier done !

File: docs/containerizer-internals.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Mesos currently supports the following containerizers:
 
 * Composing
 * [Docker](docker-containerizer.md)
-* [Mesos](containerizer.md)
+* [Mesos](containerizers.md)
```
Comment (bbannier) [inline docs/containerizer-internals.md]: Ups, this one should in fact have pointed to `mesos-containerizer.md`.

Comment (conradkdotcom) [inline docs/containerizer-internals.md]: woops, me just commit that once again

**Comment by conradkdotcom** [comment]
@bbannier how about now ?

**Pull request closed without merging**